probe/host/mem.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package host
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+// Mem is the resource usage for memory and disk
+type Mem struct {
+	ResourceUsage `yaml:",inline"`
+	Threshold     float64 `yaml:"threshold"`
+}
+
+// Command returns the command to get the cpu usage
+func (m *Mem) Command() string {
+	return `free -m | awk 'NR==2{printf "%s %s %.2f\n", $3,$2,$3*100/$2 }'`
+}
+
+// OutputLines returns the lines of command output
+func (m *Mem) OutputLines() int {
+	return 1
+}
+
+// Config returns the config of the cpu
+func (m *Mem) Config(s *Server) error {
+	if s.Threshold.Mem == 0 {
+		s.Threshold.Mem = DefaultMemThreshold
+		log.Debugf("[%s / %s] Memory threshold is not set, use default value: %.2f", s.ProbeKind, s.ProbeName, s.Threshold.Mem)
```

Comment:
```suggestion
		log.Debugf("[%s / %s] Memory threshold is not set, using default value: %.2f", s.ProbeKind, s.ProbeName, s.Threshold.Mem)
```

Reply:
it looks like this suggestion is missing